### ISSUE TITLE
Update Bundler to Bundler 2

### ISSUE THREAD

**Issue by nemodjuric** [department-of-veterans-affairs/caseflow#9270]
Bundler 2 came out on Jan 3 2019 and Bundler team has one year deprecation warning period after which support for old version stops. Updating bundler means that if gems were packaged using one version locally then they have to be packaged with the same version during deployment. 
Also, there's CircleCI configuration part that might need a change (Travis for efolder and feedback apps). 

Relevant links:
https://bundler.io/blog/2019/01/03/announcing-bundler-2.html

**Comment by lisac-usds** [comment]
possibly relevant: in Dec 2018 we did some work that would have the side effect of keeping us on `bundler`  `1.16`. Might come up in the course of resolving this Issue.

Explanation for the Dec 2018 work: we were seeing a strange error thrown during db migrations that we associated with `bundler 1.17`; so we wired things up so that `1.17` would not get pulled in.  
Details in [appeals-deployment#1822](https://github.com/department-of-veterans-affairs/appeals-deployment/pull/1822) and [slack](https://dsva.slack.com/archives/C2ZAMLK88/p1545260646250300).
